CefSharp.Core/Internals/CefBrowserHostWrapper.cpp

```diff
@@ -336,12 +336,30 @@ void CefBrowserHostWrapper::ImeSetComposition(String^ text, cli::array<Compositi
     _browserHost->ImeSetComposition(StringUtils::ToNative(text), underlinesVector, CefRange(), range);
 }
 
+void CefBrowserHostWrapper::ImeSetComposition(
```

Comment:
These can likely be removed and you just get a reference to the underlying `CefBrowserHost` rather than adding code.

It's my understanding that some of the params are only used on `Mac` and can be left out of any `Windows` implementation.